SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialLoadBalanceEnforcer.cpp

```diff
@@ -136,3 +204,22 @@ TArray<SpatialLoadBalanceEnforcer::AclWriteAuthorityRequest> SpatialLoadBalanceE
 
 	return PendingRequests;
 }
+
+void SpatialLoadBalanceEnforcer::QueueAclAssignmentRequest(const Worker_EntityId EntityId)
+{
+	UE_LOG(LogSpatialLoadBalanceEnforcer, Verbose, TEXT("Queueing ACL assignment request for entity %lld on worker %s."), EntityId, *WorkerId);
+	AclWriteAuthAssignmentRequests.Add(WriteAuthAssignmentRequest(EntityId));
+}
+
+bool SpatialLoadBalanceEnforcer::AclAssignmentRequestIsQueued(const Worker_EntityId EntityId)
+{
+	return AclWriteAuthAssignmentRequests.ContainsByPredicate([EntityId](const WriteAuthAssignmentRequest& Request) { return Request.EntityId == EntityId; });
+}
+
+bool SpatialLoadBalanceEnforcer::CanEnforce(Worker_EntityId EntityId)
+{
+	EntityAcl* Acl = StaticComponentView->GetComponentData<EntityAcl>(EntityId);
```

Comment:
Can we just use `HasComponent` for each of these instead?

Reply:
Also for the ACL you can also just do the `HasAuthority(...)` which implicitly checks the existence of the data

Reply:
done, but kept both the data and authority check to still be explicit